docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# First stage: build the application
+FROM golang:1.13.0-alpine AS builder
+ARG GOARCH
+ARG GOARM
+
+WORKDIR /go/src/github.com/CESARBR/knot-babeltower
+COPY . .
+
+# Install build utilities and glide dependency manager
+RUN apk add --update --no-cache --virtual .build-deps \
+    make \
+    git \
+    curl
+RUN curl https://glide.sh/get | sh
+
+# Install project development tools and dependencies
+RUN go get github.com/ahmetb/govvv
+RUN make deps
```

Comment:
We could use `go mod` instead :smile:

Considering we the `COPY` command in line 7, we dont need to copy go mod and sum files :grimacing:
 
Heres a suggestion :point_down: 

 ```suggestion

# Download project dependencies
RUN go mod download
RUN go mod verify
```

Reply:
Oh, nice. I didn't know the GOPROXY, thanks!
As it became the default, I prefer to use it.